### ISSUE TITLE
accessibility: fix accessibility issues where nulls and empty strings are passed to components

### ISSUE THREAD

**Issue by hollandjg** [thepolicylab-projectportals/project-portal-theme#642]
Fix places where if an input is empty, we get an accessibility violation.

_Originally posted by @hollandjg in https://github.com/thepolicylab-projectportals/project-portal-theme/issues/641#issuecomment-1758152437_
